test/parse/test.sh

```
@@ -1,22 +1,32 @@
 )
 
+make
 for file in "${!test_cases[@]}"; do
     expected_status="${test_cases[$file]}"
     ./parse "$file" 2> /dev/null
     if [ "$?" -eq "$expected_status" ]; then
-        echo -e "\e[32mOK\e[0m"
+        echo -e "$file: \e[32mOK\e[0m"
     else
-        echo -e "\e[31mNG (got $status)\e[0m"
+        echo -e "$file: \e[31mNG (got $status)\e[0m"
```

Shellcheck (0.10.0)

[warning] status is referenced but not assigned.

(SC2154)